style.css

```diff
@@ -481,6 +464,336 @@ textarea::placeholder {
   color: #172b4d;
 }
 
```

Comment:
### contact section desktop version


- Hey, let's align the content of this section on the desktop version to be close to the Figma design to make our design consistent. see the images below to improve your design:

*Your design*
![image](https://user-images.githubusercontent.com/59397159/236646285-56063b61-a40d-4cdd-a02c-a5ed9b796949.png)


*Figma design*
![image](https://user-images.githubusercontent.com/59397159/236646340-29bd9c4e-0fc2-4004-92f5-5002f4012954.png)